README.md

```diff
@@ -1,344 +1,41 @@
 
-### Generate new words
+**Hathor headless wallet** is the official headless wallet of Hathor. It is intended for use by external systems that integrate with Hathor Network, specifically for managing their wallets.
 
-This script generates a new seed with 24 words. It is useful when you'd like to start a brand new wallet. The seed is a sensitive data that cannot be shared with anyone else. Remember to clean up your clipboard after copying & pasting.
+## Operation and usage
 
-```bash
-$ npm run generate_words
+To know how to operate and use Hathor headless wallet, see [Hathor headless wallet at Hathor docs — official technical documentation of Hathor](https://docs.hathor.network/pathways/components/headless-wallet).
 
-work above economy captain advance bread logic paddle copper change maze tongue salon sadness cannon fish debris need make purpose usage worth vault shrug
-```
+## Support
 
-### Evaluate API Docs
+If after consulting the documentation, you still need **help to operate and use Hathor headless wallet**, [send a message to the `#development` channel on Hathor Discord server for assistance from Hathor team and community members](https://discord.com/channels/566500848570466316/663785995082268713).
 
-This script generates a fully interactive webpage containing the documentation of the API. It is useful to check if the documentation is correct and up-to-date in a human friendly interface.
+If you observe an incorrect behavior while using Hathor headless wallet, we encourage you to [open an issue to report this failure](https://github.com/HathorNetwork/hathor-wallet-headless/issues/new).
 
-This feature requires NodeJS v16 or greater and the [`redocly/cli`](https://github.com/Redocly/redocly-cli) library installed globally:
-```bash
-# Validate the generated docs according to Redocly rules and visualize them
-npm run docs
-```
+You can also [open an issue to request a new feature you wish to see](https://github.com/HathorNetwork/hathor-wallet-headless/issues/new).
 
-For a machine-readable version of the API docs, start the Headless wallet and use the `[GET] /docs` endpoint.
+Please do not open an issue to report a security breach or vulnerability. Instead, follow the guidelines described at [SECURITY](SECURITY.md) for safely reporting, fixing and disclosing security issues.
 
-## Multisig
+## Tests
 
-Check out how to configure and use as a MultiSig wallet on [MULTISIG.md](./MULTISIG.md)
+- [Testing guidelines](/docs/testing.md)
+- [QA: suggested test sequence](/docs/QA.md)
 
-## Testing
+## Contributing
 
-This application has two different contexts for testing.
-Refer to [TESTING.md](./TESTING.md) for more information on this.
+To contribute to the development of Hathor headless wallet, we encourage you to fork the `master` branch, implement your code, and then [open a pull request to merge it into `master`, selecting the "feature branch template"](https://github.com/HathorNetwork/hathor-wallet-headless/pulls).
```

Comment:
I think we could remove this link, it just goes to the list of open pull requests. What do you think?

Reply:
Change made:
- The link was wrong. The link was pointing to the list of PRs, when it should actually point to the new PR opening page.
- Regarding the usefulness and relevance of this link, I agree that even after the change, it is not an indispensable link.
- We could remove it, but I think it helps to serve as a "call to action", in a discreet but relevant way for us to point out that we encourage and suggest collaborations from the entire community.